frontend/tsconfig.json

```diff
@@ -6,6 +6,17 @@
     "module": "ESNext",
     "skipLibCheck": true,
 
+    "paths": {
```

Comment:
we will need to add a new path each time a new folder is added (and if we forget, it would fallback to relative paths which is bad).

I think we could actually just add this rather than paths:
```
"baseUrl": "."
```

Then you would import like this:
```
import toast from "components/Toast";
```

rather than:
```
import toast from "@components/Toast";
```